### PR TITLE
feat(console): init

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

init console project with parceljs

## Testing

image

image

### CODE REVIEW

File: packages/console/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "@types/react": "^17.0.14",
+    "@types/react-dom": "^17.0.9",
+    "eslint": "^8.1.0",
+    "lint-staged": "^11.1.1",
+    "parcel": "^2.3.1",
+    "postcss": "^8.4.6",
+    "postcss-modules": "^4.3.0",
+    "prettier": "^2.3.2",
+    "stylelint": "^13.13.1",
+    "typescript": "^4.3.5"
```
Comment (demonzoo) [inline packages/console/package.json:35]: Should we use typescript `4.5.5`? It's been there for a while now

Comment (gao-sun) [reply packages/console/package.json:36]: let's upgrade the workspace typescript version in a separate PR

Comment (demonzoo) [reply packages/console/package.json:36]: Sure, np

**Review by demonzoo — APPROVED** [review]
LGTM

**Pull request merged**